.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+
+# This workflow can be manually triggered to create a GitHub release and upload release bins.
+#
+# Steps to use:
+# - Create a new Git tag and push
+# - Navigate to https://github.com/rmohr/bazeldnf/actions/workflows/release.yml
+# - Click on "Run workflow"
+# - Select the tag you want to release
+# - Click on green button
```

Comment:
This is definitely a great first step.

Do you think we also need the following steps?


1. add the binary to a tag (what's done here)
2. push a commit to main which will reference the latest tag with the right digests of the binaries
3. create a draft-release which contains the right import snippet in the text

I think for simplicity it is ok to assume that we do always only consecutive releases on main and don't branch.

Reply:
Good idea. This elegantly solves the version drift problem. We might want to build artifacts and publish hashes before we tag though. Otherwise, importing bazeldnf via git_repository and tag yields counter-intuitive behavior (binary would be from an earlier version)

Reply:
> We might want to build artifacts and publish hashes before we tag though.

hm, that is tricky but may be doable:

 1. build the binaries to get the shasum
 2. create a ci-job local commit on main where we are properly guessing the final download url and fill in the digest
 3. push this commit
 4. tag main
 5. upload